pkgs/by-name/py/pyfa/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  version = "2.60.1";
+in
+appimageTools.wrapType1 {
```

Very minor nit, feel free to ignore. Today both `wrapType1` and `wrapType2` do the exact same thing, `wrapType1` is only kept for backwards compatibility.
```suggestion
appimageTools.wrapType2 {
```

---

At least according to the tool it say you have to choose the appropriate one, and I started with Type2 and at the time this didn't work. But maybe there has been changes to the tool since end of may

https://ryantm.github.io/nixpkgs/builders/images/appimagetools/

---

Weird that it didn't work for you, it points to the same since [2020](https://github.com/NixOS/nixpkgs/commit/49e7871196ea43f6b8b6c81801f632f57ab892de). I'm not sure why the documentation you linked is that outdated, but the current version of that says [the two were unified](https://github.com/NixOS/nixpkgs/blob/master/doc/build-helpers/images/appimagetools.section.md#wrapping-ssec-pkgs-appimagetools-wrapping).

---

i guess that is the better documentation :D